app/models/invoice.rb

```diff
@@ -1,6 +1,7 @@
 class Invoice < ApplicationRecord
   belongs_to :customer
   belongs_to :merchant
+  has_many :transactions
```

Comment:
because transaction belongs_to invoice, I added this relationship. Still not sure about the many part though. Why would a transaction have multiple invoices?